src/platform/item/emote/emote.ts

```diff
@@ -0,0 +1,60 @@
+import { isThirdParty } from '..'
+import { generateValidator, JSONSchema } from '../../../validation'
+import {
+  BaseItem,
+  baseItemProperties,
+  requiredBaseItemProps
+} from '../base-item'
+import { standardProperties, StandardProps } from '../standard-props'
+import { thirdPartyProps, ThirdPartyProps } from '../third-party-props'
+import { EmoteDataADR73 } from './adr73/emote-data-adr73'
+
+export type EmoteADR73 = BaseItem &
+  (StandardProps | ThirdPartyProps) & { emoteDataADR73: EmoteDataADR73 }
+
+/** @alpha */
+export type Emote = EmoteADR73
+
+/** @alpha */
+export namespace Emote {
+  export const schema: JSONSchema<Emote> = {
+    type: 'object',
+    properties: {
+      ...baseItemProperties,
+      ...standardProperties,
+      ...thirdPartyProps,
+      emoteDataADR73: EmoteDataADR73.schema
+    },
+    additionalProperties: true,
+    required: [...requiredBaseItemProps],
+    oneOf: [
```

Comment:
The error reporting of oneOf is not very nice and barely useful. I'd suggest adding `errorMessage` property like the one below.

Reply:
👍